person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(name, age, parent_permission: true)
```

Comment:
- [ ] Kindly revert to the changes you had earlier. I cannot understand why you changed up everything when you were asked to fix only 1 line of code. The `name` should be an optional parameter.

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
  ```